stable/velero/templates/backupstoragelocation.yaml

```diff
@@ -0,0 +1,47 @@
+{{- $root := . }}
+{{- with .Values.configuration }}
+{{- with .backupStorageLocation }}
+apiVersion: velero.io/v1
+kind: BackupStorageLocation
+metadata:
+  name: default
+  labels:
+    chart: {{ template "velero.chart" $root }}
+    heritage: {{ $root.Release.Service }}
+    release: {{ $root.Release.Name }}
+    app: {{ template "velero.name" $root }}
+spec:
+  provider: {{ .name  }}
+  objectStorage:
+    bucket: {{ .bucket  }}
+    {{- with .prefix }}
+      prefix: {{ . }}
```

Comment:
This should be de-dented, see PR #11430.

Reply:
Done.